java-bootcamp-bahia-t1/src/ItemManager.java

```diff
@@ -0,0 +1,6 @@
+/* For add or remove products or offers from the Container */
+public class ItemManager {

```

Comment:
If this class was created thinking on the final project it shouldn't be uploaded to the repo for this topic